### ISSUE TITLE
Select & Combobox: Screen reader doesn't announce selected item in dropdown list.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2242]
**Describe the bug**
The screen reader doesn't announce the selected item in the dropdown list.

**Expected behavior**
The screen reader should announce the selected item in the dropdown list.

**Additional context**
[Slack message](https://cengage.slack.com/archives/C8Y4FR63U/p1770886286561789),
[Details](https://cengage.slack.com/archives/C8Y4FR63U/p1775565799880079)

<img width="468" height="388" alt="Image" src="https://github.com/user-attachments/assets/3f7bf819-84a4-4edc-958b-b47a6925152f" />

**Comment by nikitaorliak-cengage** [comment]
**Tested on:**
- V4:
https://storybook-preview-v4-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/select--default
https://next--upbeat-sinoussi-f675aa.netlify.app/api/select/#custom_components

- V5: 
https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/select--default
https://rc--upbeat-sinoussi-f675aa.netlify.app/api/select/#custom_components

Everything works as expected. No bugs found.

**Comment by silvalaura** [comment]
This ticket is closed but in progress. Just wanted to check what status it should be in?

**Comment by olehnoskov-cengage** [comment]
@silvalaura 
I reopened this ticket, because of this:

On ComboBox component Screen reader does not announce “selected” state of element.
 https://infoseci.atlassian.net/browse/AAGL-1494

https://cengage.slack.com/archives/C8Y4FR63U/p1775565799880079

**Comment by olehnoskov-cengage** [comment]
The Infosec team retested this ticket, and the issue was resolved.
